rapid7_insightvm/komand_rapid7_insightvm/util/resource_helper.py

```diff
@@ -71,13 +37,15 @@ class ResourceHelper(object):
     the __init__ function during instantiation.
     """
 
-    _ERRORS = {
-        400: "Bad Request",
-        401: "Unauthorized",
-        404: "Not Found",
-        500: "Internal Server Error",
-        503: "Service Unavailable",
-        000: "Unknown Status Code"
+    # Currently only handling the most common requests exceptions more can be added as needed
+    Request_exceptions = {
```

Comment:
```suggestion
    REQUEST_EXCEPTIONS = {
```
All caps for constants